scripts/check_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,269 @@
+                {
+                    'name': test_suite_name,
+                    'module': test_suite_module
+                }
+            )
+    return sorted(e2e_test_suites, key=lambda x: x['name'])
+
+
+def get_acceptance_test_suites_from_acceptance_directory() -> List[TestSuiteDict]: # pylint: disable=line-too-long
+    """Gets the acceptance test suites from the acceptance test 
+    specs directory.
+    """
+
+    acceptance_test_files = glob.glob(
+        os.path.join(ACCEPTANCE_TEST_SPECS_DIRECTORY, '**/*.spec.ts'))
+    acceptance_test_suites: List[TestSuiteDict] = [
+        {
+            'name': os.path.relpath(
+                module,
+                ACCEPTANCE_TEST_SPECS_DIRECTORY
+            ).replace('.spec.ts', ''),
+            'module': os.path.relpath(module, os.getcwd())
+        } for module in acceptance_test_files
+    ]
+    return sorted(acceptance_test_suites, key=lambda x: x['name'])
+
+
+def compute_test_suites_difference(
+    test_suites_from_config: List[TestSuiteDict],
+    test_suites_from_directory: List[TestSuiteDict]
+) -> List[TestSuiteDict]:
+    """Computes the difference between the test suites from the CI config files
+    and the test suites from the test files.
+
+    Args:
+        test_suites_from_config: list(dict). A list of dictionaries sorted
+            by name, each containing the name and module of a test suite from
+            the CI config files.
+        test_suites_from_directory: list(dict). A list of dictionaries sorted 
+            by name, each containing the name and module of a test suite from
+            the test files.
+
+    Returns:
+        list(dict). A list of test suites which are not present in both 
+        the CI config files and the test files, but are present in
+        either of them.
+    """
+    test_suites_difference = [
+        test_suite for test_suite in test_suites_from_directory +
+        test_suites_from_config if test_suite not in test_suites_from_config or
+        test_suite not in test_suites_from_directory
+    ]
+    return test_suites_difference
+
+
+def main() -> None:
+    """Checks that the CI config files and the test files have 
+    the same test suites.
+    """
+
+    print(
+        'Checking all acceptance test suites are captured in CI '
+        'config files...'
+    )
+    acceptance_test_suites_from_config = (
+        get_acceptance_test_suites_from_ci_config_file())
+    acceptance_test_suites_from_directory = (
+        get_acceptance_test_suites_from_acceptance_directory())
+    acceptance_test_suites_difference = compute_test_suites_difference(
+        acceptance_test_suites_from_config,
+        acceptance_test_suites_from_directory
+    )
+
+    if len(acceptance_test_suites_difference) > 0:
+        raise Exception(
+            'Acceptance test suites and CI test suites are not in sync. '
+            'The following suites are not in sync: %s' % (
+                acceptance_test_suites_difference)
+        )
```

Comment:
Should there be instructions on what to do?

Reply:
ditto for errors below

Reply:
Done.